docs/apache-airflow/howto/docker-compose/docker-compose.yaml

```diff
@@ -58,6 +58,7 @@ x-airflow-common:
     AIRFLOW__CELERY__RESULT_BACKEND: db+postgresql://airflow:airflow@postgres/airflow
     AIRFLOW__CELERY__BROKER_URL: redis://:@redis:6379/0
     AIRFLOW__CORE__FERNET_KEY: ''
+    AIRFLOW__WEBSERVER__SECRET_KEY: 'replace_me_by_valid_key'
```

Comment:
I would also add the comment about importance of change this value. I guess it still be a point that in production after that a lot of user would have this as default value, the same as `airflow`:`airflow` as admin user.

I'm not sure why it is never been there, maybe some reason for that, but it is unknown for me. 

And maybe also a good idea to add a link to the configuration reference.

BTW every non empty key could be considered a valid secret key.

Reply:
Yes. We cannot add it this way. We used to have similar key in Airflow config by default and it has been reported and fixed as security vulnerability (and fixed by removing the key) - precisely because people did not change it in production when they used our default settings. They might usethe docker compose as starting point . I think we should instead  update instructions in https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html on how to generate random key

Reply:
https://www.cybersecurity-help.cz/vdb/SB2020122314

Reply:
And that is the answer why it not there 🤣 .

So that mean we should add a comment in docker-compose: "Do not forget to add the key: <link to the doc reference>"

Also might be a good idea to add info into the https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html, like add `AIRFLOW__WEBSERVER__SECRET_KEY` with a "strong" value which you could generate for example by run (source: https://unix.stackexchange.com/a/230676)

```console
tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_`{|}~' </dev/urandom | head -c 63; echo
```

The same maybe valid for Fernet key I do not know how many users of this docker-compose run with an empty Fernet key